aeternity/config.py

```diff
@@ -0,0 +1,76 @@
+import os
+
+import requests
+
+
+class ConfigException(Exception):
+    pass
+
+class Config:
+    default_config = None
+
+    def __init__(self, local_port=None, internal_port=None, websocket_port=None, host='localhost'):
+        try:
+            if local_port is None:
+                local_port = os.environ.get('AE_LOCAL_PORT', 3013)
+            self.local_port = local_port
+            if internal_port is None:
+                internal_port = os.environ.get('AE_LOCAL_INTERNAL_PORT', 3113)
+            self.local_internal_port = internal_port
+            if websocket_port is None:
+                websocket_port = os.environ.get('AE_WEBSOCKET', 3114)
+            self.websocket_port = websocket_port
+        except KeyError:
+            raise ConfigException(
+                'You must either specify the Config manually, use '
+                'Config.set_default or provide the env vars AE_LOCAL_PORT, '
+                'AE_LOCAL_INTERNAL_PORT and AE_WEBSOCKET'
+            )
+
+        self.websocker_url = f'ws://{host}:{self.websocket_port}/websocket'
```

Comment:
websocker is my favorite of all e-sports :)

Reply:
Mine too!